consensus/service/src/consensus_service.rs

```diff
@@ -382,12 +382,16 @@ impl<
                 .build(),
         );
 
-        let server_builder = ServerBuilder::new(env)
+        let server_builder = ServerBuilder::new(env.clone())
             .register_service(client_service)
             .register_service(blockchain_service)
             .register_service(health_service)
             .register_service(attested_service)
             .register_service(build_info_service)
+            .channel_args(
+                <ServerBuilder as ConnectionUriGrpcioServer>::default_channel_builder(env)
+                    .build_args(),
+            )
```

Comment:
This can now be `set_default_channel_args`
```suggestion
            .set_default_channel_args(env)
```